test/fork/migration-v2.js

```diff
@@ -879,95 +885,85 @@ describe('V2 upgrade', function () {
 
     const expectedPoolConfigurations = {};
     expectedPoolConfigurations[foundationPoolId] = {
-      maxFee: 20, // TODO: waiting for final value for maxPoolFee - sheet has 100, we currently store 20
+      maxFee: 99,
       initialFee: 0,
       manager: FOUNDATION,
       isPrivatePool: true,
+      // stakingNFTId: stakingNFTSupplyBefore.add(1),
     };
     expectedPoolConfigurations[hughPoolId] = {
       maxFee: 20,
       initialFee: 10,
       manager: HUGH,
       isPrivatePool: false,
+      stakingNFTId: stakingNFTSupplyBefore.add(1),
```

Comment:
i was finding the ++ approach cleaner, because you can move things around easier, but this is fine too